include/fastdds/rtps/transport/TCPTransportDescriptor.h

```diff
@@ -19,14 +19,14 @@
 #include <fastrtps/fastrtps_dll.h>
 #include <iostream>
 
-namespace eprosima{
-namespace fastdds{
-namespace rtps{
+namespace eprosima {
+namespace fastdds {
+namespace rtps {
 
 /**
-* Transport configuration
-* @ingroup TRANSPORT_MODULE
-*/
+ * Transport configuration
+ * @ingroup TRANSPORT_MODULE
+ */
 typedef struct TCPTransportDescriptor : public SocketTransportDescriptor
 {
     struct TLSConfig
```

Comment:
We could use the rule of 0 on this struct, just by adding default values to `options`, `verify_mode` and `handshake_role`